File: week-07/day-3/src/main/java/com/greenfoxacademy/springstart/Greeting.java
package com.greenfoxacademy.springstart;

import java.util.concurrent.atomic.AtomicLong;

public class Greeting {
  private AtomicLong id;
  private String content;

  Greeting(AtomicLong id, String content) {
    this.id = id;
    this.content = content;
  }

  public AtomicLong getId() {
    return id;
  }

  public void setId(AtomicLong id) {
    this.id = id;
  }

  public String getContent() {
    return content;
  }

  public void setContent(String content) {
    this.content = content;
  }
}

File: mypractice-C#/AspDotNetIntroApp/AspDotNetIntroApp/Models/Greeting.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AspDotNetIntroApp.Models
{
    public class Greeting
    {
        public long Id;
        public String Content;

        public Greeting()
        {

        }

        public Greeting(long id, string content)
        {
            Id = id;
            Content = content;
        }

        public long GetId()
        {
            return this.Id;
        }

        public void SetId(long id)
        {
            this.Id = id;
        }

        public String GetContent()
        {
            return this.Content;
        }

        public void SetContent(String content)
        {
            this.Content = content;
        }
    }
}
